lib/vpqr.js

```diff
@@ -0,0 +1,91 @@
+/*!
+ * Copyright (c) 2021 Digital Bazaar, Inc. All rights reserved.
+ */
+const cborld = require('@digitalbazaar/cborld');
+const qrcode = require('@nuintun/qrcode');
+import citContext from 'cit-context';
+
+// import * as baseN from './base-n.js';
+import base32Encode from 'base32-encode';
+import base32Decode from 'base32-decode';
+
+const {
+  Encoder,
+  // QRByte,
+  QRAlphanumeric,
+  // ErrorCorrectionLevel
+} = qrcode;
+
+const {appContextMap: citAppContextMap} = citContext;
+
+const VP_QR_VERSION = 'VP1';
+const base32uppercase = 'B'; // multibase prefix
```

Comment:
I'd like to see this as `BASE_32_UPPER_CASE_MULTIBASE_PREFIX` or similar to improve readability below.

Reply:
Renamed, thanks.